addOns/ascanrules/ascanrules.gradle.kts

```diff
@@ -32,7 +32,7 @@ zapAddOn {
                 dependencies {
                     addOns {
                         register("custompayloads") {
-                            version.set(">= 0.9.0 & < 1.0.0")
+                            version.set(">= 0.15.0 & < 1.0.0")
```

Comment:
Update the existing entry to also mention this add-on, while optional we should not release features that will not work.